cli/cmd/plugin/diagnostics/go.mod

```diff
@@ -2,9 +2,12 @@ module github.com/vmware-tanzu/community-edition/cli/cmd/plugin/diagnostics
 
 go 1.16
 
+replace github.com/vmware-tanzu/community-edition => ../../../../
```

Comment:
@stmcginnis at every instance of this, would  you mind putting a comment for why we import ourselves and do this? Ty!

Reply:
Done!